hardhat.config.ts

```diff
@@ -17,6 +17,37 @@ task("accounts", "Prints the list of accounts", async (args, hre) => {
   }
 });
 
+task("compile", "Compiles the entire project, building all artifacts", async (args, hre, runSuper) => {
+  await runSuper(args);
+
+  const fs = require('fs');
+
+  // File destination.txt will be created or overwritten by default.
+  await new Promise<void>((resolve, reject) => {
+    fs.copyFile('./constants/mainnet-constants.json', './frontend/src/hardhat/mainnet-constants.json', (err: Error) => {
+      if (err) reject(err);
+      console.log('mainnet-constants.json was copied to the frontend');
+      resolve()
+    });
+  })
+
+  await new Promise<void>((resolve, reject) => {
+    fs.copyFile('./constants/goerli-constants.json', './frontend/src/hardhat/goerli-constants.json', (err: Error) => {
```

Comment:
goerli-constants now is completely deleted. Does this not throw an error then?

Reply:
When hardhat compile is run now, the constants get copied from the ./constants into the hardhat folder (all binaries).

It will throw an error, if compilation has not been done.

Reply:
cool